src/index.ts

```diff
@@ -46,3 +56,14 @@ export const Ludex = {
   OrganizationScoped,
   ClientScoped,
 };
+
+export {
+  Chain,
+  PayoutType,
+  PayoutState,
+  Environment,
+  ChallengeState,
+  RedeemType,
+  AxiosError,
+  ZodError
```

Comment:
Export AxiosError and ZodError for user to construct their error handling logics